lib/flowLSP/utils/assertFlowSupportsLSP.js

```diff
@@ -4,7 +4,7 @@ import semver from 'semver';
 const FLOW_VERSION_FOR_LSP = '>=0.75';
 
 export default function assertFlowSupportsLSP(version: string) {
-  if (!semver.satisfies(version, FLOW_VERSION_FOR_LSP)) {
+  if (!semver.satisfies(version.replace(/-rc/gu, ''), FLOW_VERSION_FOR_LSP)) {
```

Comment:
You need to put the same fix here also https://github.com/flowtype/flow-for-vscode/blob/7bd24accc13cc842c2ff6fce18e37d61b66b53d7/lib/flowLSP/FlowLanguageClient/createMiddleware.js#L25-L28
To avoid logic duplication create separate function in utils and use that function in both places and also add test case for it.

Reply:
use `semver.satisfies(version, FLOW_VERSION_FOR_LSP, { includePrerelease: true })`